Makefile

```diff
@@ -27,3 +30,6 @@ drop: ## Drops database. Recreate with `up` then `admin`
 
 admin: up ## Launches admin site in default browser
 	open http://localhost:8000/wp-admin
+
+network:
```

Comment:
adding a comment like `## sets up the multisite network` will allows this to be documented more clearly when running `make` with no command (I know this is not an actual PR yet but while I'm here I thought I'd say it :)